### ISSUE TITLE
help：Sizes of tensors must match except in dimension 2. Expected size 416 but got size 64 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by tiandaoyuxi** [MrForExample/ComfyUI-3D-Pack#104]
Error occurred when executing [Comfy3D] TripoSR:

Sizes of tensors must match except in dimension 2. Expected size 416 but got size 64 for tensor number 1 in the list.

File "D:\ComfyUI_TEST\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_TEST\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_TEST\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_TEST\python_embeded\Lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_TEST\ComfyUI\custom_nodes\ComfyUI-3D-Pack\nodes.py", line 1653, in run_TSR
image = torch.cat((image, mask), dim=2).detach().cpu().numpy()
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by MrForExample** [comment]
Hi @tiandaoyuxi 
This is a same issue as [issue 99](https://github.com/MrForExample/ComfyUI-3D-Pack/issues/99)
